### PR TITLE
{tools}[GCC/4.6.3] attr v2.4.47

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2074.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8bb4fbc0f73a57326107 for a full test report.

File: easybuild/easyconfigs/a/attr/attr-2.4.47-GCC-4.6.3.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+easyblock = 'ConfigureMake'
+
+name = 'attr'
+version = '2.4.47'
+
+homepage = 'http://savannah.nongnu.org/projects/attr'
+description = """Commands for Manipulating Filesystem Extended Attributes"""
+
+toolchain = {'name': 'GCC', 'version': '4.6.3'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/a/attr/attr-2.4.47-GCC-4.6.3.eb]: old?

File: easybuild/easyconfigs/a/attr/attr-2.4.47-GCC-4.6.3.eb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+source_urls = [GNU_SAVANNAH_SOURCE]
+sources = ['attr-%(version)s.src.tar.gz']
+
+installopts = 'install-lib'
+
+sanity_check_paths = {
+    'files': ['bin/attr', 'bin/getfattr', 'bin/setfattr', 'lib/libattr.so.1', 'lib/libattr.so.1.1.0'],
```
Comment (wpoely86) [inline easybuild/easyconfigs/a/attr/attr-2.4.47-GCC-4.6.3.eb]: just check the `.so` and use the constant

File: easybuild/easyconfigs/a/attr/attr-2.4.47-GCC-4.6.3.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/a/attr/attr-2.4.47-GCC-4.6.3.eb]: yeah, needed to fix issue with old easyconfig, cfr. https://github.com/hpcugent/easybuild-easyconfigs/pull/2699 and https://github.com/hpcugent/easybuild-easyblocks/pull/854

File: easybuild/easyconfigs/a/attr/attr-2.4.47-GCC-4.6.3.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/a/attr/attr-2.4.47-GCC-4.6.3.eb]: there is no `.so`, only these, so the constant doesn't apply here imho

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2074.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e2119d7cc123ac91b403 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2074.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/760d11264a81bfd904ff for a full test report.

**Comment by besserox** [comment]
@boegel What about additional sanity checks on `include/attr/libattr.h` and `include/attr/xattr.h` ?

**Comment by boegel** [comment]
@besserox: well, I'm checking for a non-empty `include/attr` directory...

I can extend that each of the files in there are present?

```
attributes.h  error_context.h  libattr.h  xattr.h
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2074.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/cee847387cdb870a8f59 for a full test report.

**Comment by boegel** [comment]
@besserox: extended sanity check paths to check all 4 include files

**Comment by boegel** [comment]
Going in, thanks for the feedback @wpoely86 and @besserox!

**Pull request merged**